chooloolib/src/main/AndroidManifest.xml

```diff
@@ -3,14 +3,10 @@
     package="com.chooloo.www.chooloolib">
 
     <uses-permission android:name="android.permission.CALL_PHONE" />
-    <uses-permission android:name="android.permission.MODIFY_PHONE_STATE" />
+    <uses-permission android:name="android.permission.ANSWER_PHONE_CALL" />
```

Comment:
You missed an `S`, it's `ANSWER_PHONE_CALLS`: ref: [`Manifest.permission.ANSWER_PHONE_CALLS`](https://developer.android.com/reference/android/Manifest.permission#ANSWER_PHONE_CALLS)

Reply:
yes you are right. I read today a other dialer app to get the idea why modify phone state is used. I only make a fast forward fix and don't get any compile error on android studio. So :)